client/memory.js

```diff
@@ -68,225 +77,104 @@ export default class Memory extends React.Component {
       headers: {
         'Authorization': 'Bearer ' + token
       }
-    }).then(function(memory) {
-      // var analyses = JSON.parse(memory['_bodyInit']).tags;
-      var analyses = JSON.parse(memory['_bodyInit']).analyses[0].tags;
-      context.setState({tags: analyses, status: 'Tags:'});
+    }).then(function(res) {
+      var memory = JSON.parse(res['_bodyInit']);
+      var microsoftTags = [];
+      var clarifaiTags = [];
+      if (memory.analyses[0] === null) {
```

Comment:
but these conditional statements allow old pictures and their tags to still be rendered as well as the new pictures